libosmscout-gpx/CMakeLists.txt

```diff
@@ -1,19 +1,20 @@
 set(HEADER_FILES
-	include/osmscout/gpx/GpxFile.h
-	include/osmscout/gpx/GPXImportExport.h
-	include/osmscout/gpx/Route.h
-	include/osmscout/gpx/Track.h
-	include/osmscout/gpx/Waypoint.h
-	include/osmscout/gpx/TrackPoint.h
-	include/osmscout/gpx/TrackSegment.h
-	include/osmscout/gpx/Utils.h
+    include/osmscout/gpx/GpxFile.h
+    include/osmscout/gpx/GPXImportExport.h
+    include/osmscout/gpx/Route.h
+    include/osmscout/gpx/Track.h
+    include/osmscout/gpx/Waypoint.h
+    include/osmscout/gpx/TrackPoint.h
+    include/osmscout/gpx/TrackSegment.h
+    include/osmscout/gpx/Utils.h
+    include/osmscout/gpx/Extensions.h
```

Comment:
Please also add the header to the meson build files. Meson would also need it for installation.

Reply:
Don't know Meson, but I will learn ...

Reply:
Meson files rather intuitive and simple in structure. Adding a source file is a simple addition to a list of files.